ui/apps/platform/src/Containers/MainPage/Header/CLIDownloadMenu.tsx

```diff
@@ -37,25 +37,39 @@ function CLIDownloadMenu({ addToast, removeToast }: CLIDownloadMenuProps): React
 
     const appLauncherItems = [
         <ApplicationLauncherItem
-            key="app-launcher-item-cli-mac"
+            key="app-launcher-item-cli-mac-amd64"
```

Comment:
It's a bit ugly that all CLI variants are stored in two places. @vjwilson WDYT? @yselkowitz can I push to your PR to clean this up?

Reply:
@misberner 
The `key` properties are an affordance for the React templating system. They just need to be unique IDs. In fact, let's not try to keep them in sync with variant in the click handler--that's gilding a lily. Just make them the type of system

For example,
* mac
* linux-amd
* linux-ppc

Reply:
Oh, I was referring to it being referenced in the other file as well.